run.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
 
 echo "PAServer Password: securepass"
-docker run --cap-add=SYS_PTRACE --security-opt seccomp=unconfined -it -e PA_SERVER_PASSWORD=securepass -p 64211:64211 -p 8082:8082 radstudio/paserver:latest
+docker run --cap-add=SYS_PTRACE --security-opt seccomp=unconfined \
```

Comment:
Looks like this is the change which was mentioned in the [other comment thread](https://github.com/Embarcadero/paserver-docker/pull/14/files/f8c26d0319947858c567fe93f6ec7cfb747ebc54#r1464989401)

Well, as far as I see now, PAServer doesn't use any TTY functionality, however, I think it will be no harm to add deleted `--tty` as well.